lib/selinuxtest.pm

```diff
@@ -93,4 +96,57 @@ sub check_category {
     }
 }
 
+sub set_sestatus {
+    my ($self, $mode, $type) = @_;
+    my $selinux_config_file = "/etc/selinux/config";
+    select_console "root-console";
+
+    # SELinux by default
+    validate_script_output("sestatus", sub { m/SELinux status: .*disabled/ });
+
+    # workaround for "selinux-auto-relabel" in case: auto relabel then trigger reboot
+    my $results = script_run("zypper --non-interactive se selinux-autorelabel");
+    if (!$results) {
+        assert_script_run("sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=8/\' /etc/default/grub");
+    }
```

Comment:
This could also simply be

```suggestion
    assert_script_run "zypper --non-interactive se selinux-autorelabel && sed -ie \'s/GRUB_TIMEOUT.*/GRUB_TIMEOUT=8/\' /etc/default/grub";
    }
```

without the need to switch between the remote bash and the local perl context. Also I wonder why you introduce a timeout with the hardcoded timeout value of 8 but ok :)

Reply:
Good suggestion. But I keep the code as the return value of `script_run("zypper --non-interactive se selinux-autorelabel")` is  on-zero on SLES atm.

Reply:
I don't know what you mean. Maybe I got the logic wrong. To turn it around you can of course use `||` instead of `&&`